src/subscription_manager/managercli.py

```diff
@@ -1414,12 +1415,14 @@ def _primary(self):
         return True
 
     def _validate_options(self):
-        if not (self.options.pool or self.options.auto or self.options.file):
-            system_exit(os.EX_USAGE, _("Error: This command requires that you specify a pool with --pool or --file, or use --auto."))
-        if self.options.pool and self.options.auto:
-            system_exit(os.EX_USAGE, _("Error: --auto may not be used when specifying pools."))
+        if self.options.pool:
+            if self.options.auto:
+                system_exit(os.EX_USAGE, _("Error: --auto may not be used when specifying pools."))
```

Comment:
Probably an oversight on my part, but do we also need a similar check when specifying pools via file?